### PR TITLE
Small updates from the Jetstream server

### PR DESCRIPTION
**Pull request by steve-gregory**
## Description
These unstaged changes were on the jetstream server and should be included before the next release.

**Highlights:**
- Bugfix: Include domain_name for non-2.0 password auth (on openstack clouds)
- Internal: Increase time limit and duration for allocation monitoring tasks.
- Bugfix: Updated 'share_image' to pass Identity (for all credentials) rather than just project_name.

## Checklist before merging Pull Requests
- [x] Reviewed and approved by at least one other contributor.

### CODE REVIEW

File: core/models/machine_request.py [hunk 1]
```diff
@@ -297,6 +297,8 @@ def get_credentials(self):
         old_admin = old_provider.get_admin_identity().get_credentials()
         if 'ex_force_auth_version' not in old_creds:
             old_creds['ex_force_auth_version'] = '2.0_password'
+        if old_creds['ex_force_auth_version'] != '2.0_password' and 'domain_name' not in old_creds:
```
Comment (julianpistorius) [inline core/models/machine_request.py]: Note: I had to read through this method a couple of times. The name (`get_credentials`) is a bit deceptive. It does quite a bit more than just 'getting'. It seems to do some clever copying and manipulation. I imagine it started out simpler.

File: service/accounts/openstack_manager.py [hunk 2]
```diff
@@ -441,30 +441,33 @@ def shared_images_for(self, image_id):
                     for member in shared_with]
         return projects
 
-    def share_image_with_project(self, glance_image, project_name):
```
Comment (julianpistorius) [inline service/accounts/openstack_manager.py]: Not part of this PR, but I noticed that lines 430 & 431 contain tabs for indentation.

Comment (julianpistorius) [inline service/accounts/openstack_manager.py]: https://github.com/steve-gregory/atmosphere/pull/6

Comment (steve-gregory) [inline service/accounts/openstack_manager.py]: :shipit:
